contracts/pcv/balancer/BalancerLBPSwapper.sol

```diff
@@ -38,9 +38,9 @@ contract BalancerLBPSwapper is IPCVSwapper, OracleRef, Timed, WeightedBalancerPo
     /// @notice the Balancer V2 Pool id of `pool`
     bytes32 public pid;
 
-    // Balancer constants for the 99:1 -> 1:99 auction
-    uint256 private constant ONE_PERCENT = 0.01e18;
-    uint256 private constant NINETY_NINE_PERCENT = 0.99e18;
+    // Balancer constants for the 95:5 -> 5:95 auction
+    uint256 private constant FIVE_PERCENT = 0.05e18;
+    uint256 private constant NINETYFIVE_PERCENT = 0.95e18;
```

Comment:
can we just make this a configurable param set at construction instead of hardcoded constants which we constantly have to change manually?

Reply:
yes ser